### DISCUSSION TITLE
v6.0.2

### DISCUSSION THREAD

**Discussion by Ghost-chu** [PBH-BTN/PeerBanHelper#364] [公告]
## 错误修复

1. 修复了 BiglyBT/Azureus(Vuze) 的 infoHash/peerId 的 Base64 结果中带有 `/` 符号时，封禁不起效果的问题，要解决此问题，除了更新 PBH 之外，也需要您升级对应的下载器插件。
2. 修复 WebUI 的 GeoIP 图表可能会被广告拦截插件拦截的问题，如果仍被拦截，请向我们报告。
3. 修复添加订阅规则时，尽管成功但仍弹出错误提示的问题

<hr /><em>This discussion was created from the release <a href='https://github.com/PBH-BTN/PeerBanHelper/releases/tag/v6.0.2'>v6.0.2</a>.</em>

**Comment by kevenlch** [comment]
为什么打不开链接

**Comment by Kamijou66** [comment]
更新之后webui打开都是白屏怎么办，软件启动加载时提示有问题，下载数据库 java exception 时出现错误，需要查看日志来修复啥的。
[latest.log](https://github.com/user-attachments/files/16724644/latest.log)

**Reply by Kamijou66** [reply]
换回5.1.0之后webui就能正常显示了

**Reply by Ghost-chu** [reply]
> 更新之后webui打开都是白屏怎么办，软件启动加载时提示有问题，下载数据库 java exception 时出现错误，需要查看日志来修复啥的。 [latest.log](https://github.com/user-attachments/files/16724644/latest.log)

版本升级后，请手动刷新浏览器缓存

挂梯子下载
